mypy/semanal.py

```diff
@@ -4296,7 +4296,7 @@ def lookup(self, name: str, ctx: Context,
             assert isinstance(b.node, MypyFile)
             table = b.node.names
             if name in table:
-                if name[0] == "_" and name[1] != "_":
+                if len(name) > 1 and name[0] == "_" and name[1] != "_":
```

Comment:
Question: why would we need to define `_` as a function in `__builtins__`? There's no such function as far as I remember.

The original issue was about a custom module, not `__builtins__`.

Reply:
It's a standard pattern: https://docs.python.org/3/library/gettext.html#gettext.install